cmd/virsnap/export.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2019 The virnsnap authors. See file "AUTHORS".
+// Licensed under the MIT License. You have obtained a copy of the License at
+// the "LICENSE" file in this repository.
+
+// Package main implements the handlers for the different command line arguments.
+package main
+
+import (
+	"path/filepath"
+
+	"github.com/joroec/virsnap/pkg/fs"
+	"github.com/joroec/virsnap/pkg/virt"
+
+	"github.com/libvirt/libvirt-go"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// outputDir is the target directory of the backup
+	outputDir = ""
+
+	// snapshot determines whether virsnap should make a new snapshot after the
+	// machine was shut down.
+	snapshot = true
```

Comment:
I feel this as a variable name is too generic as it can mean everything (especially in a tool that deals with snapshots). Maybe we can rename it to something like `snapshotOnShutdown`?